src/js/components/explorer/detail/DetailContent.jsx

```diff
@@ -206,15 +209,23 @@ export default class DetailContent extends React.Component {
                 data={this.props.data}
                 goDeeper={this.props.goDeeper}
                 changeSubdivisionType={this.props.changeSubdivisionType}
+                goToUnreported={this.props.goToUnreported}
                 showTooltip={this.props.showTooltip}
                 hideTooltip={this.props.hideTooltip} />
         );
+
         if (this.props.total === 0 || this.props.total === null) {
             const currentIndex = this.props.trail.length - 1;
             visualizationSection = (
                 <NoAwardsScreen
                     rewindToFilter={this.props.rewindToFilter}
                     currentIndex={currentIndex} />);
+        } else if (this.props.data.get(0).id === null) {
```

Comment:
To be more specific/cover more cases, should we check for `this.props.data.size === 1` &&  `name === 'Unreported Data'` instead?